composer.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "ubmannheim/publist4ubma2",
```

Comment:
Created an issue to discuss this further: https://github.com/UB-Mannheim/publist4ubma2/issues/18